### PR TITLE
Issue 94 - Show incomplete form when user has missing data

### PR DESCRIPTION
**Pull request by ahmed-shehata**
Issue #94 Show incomplete form when user has missing data (Payment/Shipping)

### CODE REVIEW

File: AtlasUI/AtlasUI/CheckoutSummaryViewController.swift [hunk 1]
```diff
@@ -15,6 +15,9 @@ final class CheckoutSummaryViewController: UIViewController, CheckoutProviderTyp
     internal let buyButton = UIButton()
     internal let connectToZalandoButton = UIButton()
     internal let shippingPrice: Float = 0
+
+    internal let shippingView = CheckoutSummaryRow()
+    internal let paymentMethodView = CheckoutSummaryRow()
     internal private(set) var customer: Customer?
     internal private(set) var checkoutViewModel: CheckoutViewModel
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryViewController.swift]: Maybe reorder these properties and group visually UI views together and other objects:

```
    internal let productImageView = UIImageView()
    internal let productNameLabel = UILabel()
    internal let purchasedObjectSummaryLabel = UILabel()
    internal let termsAndConditionsButton = UIButton()
    internal let paymentSummaryTableview = UITableView()
    internal let stackView: UIStackView = UIStackView()
    internal let buyButton = UIButton()
    internal let connectToZalandoButton = UIButton()
    internal let shippingView = CheckoutSummaryRow()
    internal let paymentMethodView = CheckoutSummaryRow()

    internal let shippingPrice: Float = 0
    internal private(set) var customer: Customer?
    internal private(set) var checkoutViewModel: CheckoutViewModel
```

Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryViewController.swift]: The best with same order like as they're created and laid out on a screen.

**Comment by bonkey** [comment]
Great result. No conditions, only simple declarative setup.

**Comment by bonkey** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

**Pull request merged**